src/docs/sdk/performance/time-to-initial-full-display.mdx

```diff
@@ -0,0 +1,80 @@
+---
+title: "Time to Initial Display/Time to Full Display"
+---
+
+We recommend implementing this feature for mobile and desktop SDKs.
+
+The current screen load transactions timings are not reliable to understand the time spent in the screen creation.
+This is because they wait for any span created in the meantime, including any file I/O or network request.
+Time to Initial Display and Time to Full Display are spans added to the screen load transactions,
+which can be used to show more accurate information on the time spent in the screen creation.
+Being spans, they don't impact on user quota.
+
+They start as soon as a new screen is created, as long as the screen load auto-generated transactions are enabled.
+
+## Time to Initial Display
+
+Time to Initial Display is a span with operation `ui.load.initial_display` which finishes automatically
+when the screen draws its first frame. It's controlled automatically by the SDK and doesn't
+require any manual intervention from the user.
+
+It's also set as a measurement of the transaction, with `time_to_initial_display` as key and the span duration as value.
+Note: The measurement is should be set only if the span finishes with an `OK`  status.
```

Comment:
`m`: How could the TTID span end with a different status?

Reply:
The case is when a new screen starts before showing the first frame
Like an Activity that is used to only start other activities

Reply:
Please add this essential piece of information to the docs.